### PR TITLE
Update s* from monorail results

### PR DESCRIPTION
**Pull request by stephenmcgruer**
See #798 for the methodology

### CODE REVIEW

File: service-workers/service-worker/META.yml [hunk 1]
```diff
@@ -216,3 +216,27 @@ links:
...
+    - product: chrome
+      url: https://crbug.com/900700
+      results:
+        - test: fetch-event-worker-timing-frame.tentative.https.html
+    - product: chrome
+      url: https://crbug.com/1052642
+      results:
+        - test: unregister-immediately-during-extendable-events.https.html
+    - product: chrome
+      url: https://crbug.com/832071
```
Comment (stephenmcgruer) [inline service-workers/service-worker/META.yml:236]: Bug was originally about the test having unstable output, but has pivoted to be about fixing the Chrome failure.

File: shape-detection/META.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
+links:
+    - product: chrome
+      url: https://crbug.com/1067533
```
Comment (stephenmcgruer) [inline shape-detection/META.yml:3]: Unrelated to all, except arguably idlharness.https.any.sharedworker.html

File: shape-detection/META.yml [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+        - test: detection-ImageBitmap.https.html
+        - test: detection-ImageData-detached.https.html
+        - test: detection-ImageData.https.html
+        - test: detection-options.https.html
+        - test: detection-security-test.https.html
+        - test: detector-same-object.https.html
+        - test: idlharness.https.any.sharedworker.html
+        - test: shapedetection-cross-origin.sub.https.html
+    - product: chrome
+      url: https://crbug.com/789511
```
Comment (stephenmcgruer) [inline shape-detection/META.yml:19]: Bug is related but implies they should be fixed; take a closer look

File: signed-exchange/subresource/META.yml [hunk 4]
```diff
@@ -0,0 +1,5 @@
+links:
+    - product: chrome
+      url: https://crbug.com/1025074
```
Comment (stephenmcgruer) [inline signed-exchange/subresource/META.yml:3]: Not clearly related

File: storage/META.yml [hunk 5]
```diff
@@ -0,0 +1,9 @@
+links:
+    - product: chrome
+      url: https://crbug.com/1000768
```
Comment (stephenmcgruer) [inline storage/META.yml:3]: Probably not related.

File: storage/META.yml [hunk 6]
```diff
@@ -0,0 +1,9 @@
...
+      results:
+        - test: idlharness.https.any.html
+    - product: chrome
+      url: https://crbug.com/166689
```
Comment (stephenmcgruer) [inline storage/META.yml:7]: Incorrect assignment

File: streams/piping/META.yml [hunk 7]
```diff
@@ -0,0 +1,5 @@
+links:
+    - product: chrome
+      url: https://crbug.com/1093862
```
Comment (stephenmcgruer) [inline streams/piping/META.yml:3]: Unclear if related, take a closer look?

File: streams/readable-byte-streams/META.yml [hunk 8]
```diff
@@ -0,0 +1,5 @@
+links:
+    - product: chrome
+      url: https://crbug.com/614302
```
Comment (stephenmcgruer) [inline streams/readable-byte-streams/META.yml:3]: Not clearly related

File: streams/transform-streams/META.yml [hunk 9]
```diff
@@ -0,0 +1,5 @@
+links:
+    - product: chrome
+      url: https://crbug.com/1093862
```
Comment (stephenmcgruer) [inline streams/transform-streams/META.yml:3]: Not clear if related

File: streams/writable-streams/META.yml [hunk 10]
```diff
@@ -0,0 +1,5 @@
+links:
+    - product: chrome
+      url: https://crbug.com/1093862
```
Comment (stephenmcgruer) [inline streams/writable-streams/META.yml:3]: Not clear if related

File: svg/animations/META.yml [hunk 11]
```diff
@@ -0,0 +1,13 @@
+links:
+    - product: chrome
+      url: https://crbug.com/985335
```
Comment (stephenmcgruer) [inline svg/animations/META.yml:3]: Generic porting test

File: svg/types/scripted/META.yml [hunk 12]
```diff
@@ -28,3 +28,10 @@ links:
           subtest: SVGGeometryElement.prototype.isPointInStroke, no arguments.
         - test: SVGGeometryElement.isPointInFill-01.svg
           subtest: SVGGeometryElement.prototype.isPointInFill, no arguments.
+    - product: chrome
+      url: https://crbug.com/636710
```
Comment (stephenmcgruer) [inline svg/types/scripted/META.yml:32]: Export-WPT-tests bug

File: svg/styling/META.yml [hunk 13]
```diff
@@ -0,0 +1,12 @@
+links:
+    - product: chrome
+      url: https://crbug.com/1056055
```
Comment (foolip) [inline svg/styling/META.yml:3]: This is a success case, WPT import bug got turned into something more specific.

**Review by foolip — APPROVED** [review]
Did some spot checking.

**Comment by foolip** [comment]
@stephenmcgruer oops this conflicts now, sorry for slow review.

**Comment by stephenmcgruer** [comment]
> @stephenmcgruer oops this conflicts now, sorry for slow review.

And it even conflicts with work someone had to do manually :P. I've pushed the branch, one interesting case is svg/path/closepath/META.yml where the script came up with a different bug than the human. I think the script is actually correct here, so I'm keeping the script's bug.

**Pull request merged**